### PR TITLE
created second web page

### PR DESCRIPTION
**Pull request by nehalskdev**

### CODE REVIEW

File: tasks-by-users/nehalskdev/02 Poject/Cehcklist.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="Vlogging">
+    <title>Checklist</title>
+</head>
+<body>  
+    <img src="Vlog kit.jpg"
```
Comment (AbbasShaikh0304) [inline tasks-by-users/nehalskdev/02 Poject/Cehcklist.html]: alt ???

File: tasks-by-users/nehalskdev/02 Poject/index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        <a href="https://www.youtube.com/c/MumbikerNikhil" target="_blank">Nikkhil</a> and <a href="https://www.youtube.com/c/FlyingBeast320" target="_blank">Flying Beast</a>
+    </p>
+</div>
+<hr>
+<div>
+    <h1>First Vlog </h1>
+        <p>The day was Jan 11 2020, My Department decided to take all of us to a very amazing location of <strong>ND Studio.</strong></p>
+        
+        <a href="https://www.youtube.com/watch?v=WTe6WRmBEiY" target="_blank"><em style="font-family: 'Times New Roman', Times, serif;">Tom Cruise in Karjat</em></a>
+        <br><img src="First Vlog Thumb.jpg"> <br>
```
Comment (AbbasShaikh0304) [inline tasks-by-users/nehalskdev/02 Poject/index.html]: alt ?

**Comment by AbbasShaikh0304** [comment]
Add alt for every image @nehalskdev

**Comment by nehalskdev** [comment]
> Add alt for every image @nehalskdev



> Add alt for every image @nehalskdev

corrected @AbbasShaikh0304 thannks

**Review by AbbasShaikh0304 — APPROVED** [review]

**Review by mmusk — APPROVED** [review]
Use prettier @nehalskdev

**Review by coder-mezab — APPROVED** [review]

**Comment by nehalskdev** [comment]
> Use prettier @nehalskdev 

Actually there is some issue with my prettier I will resolve it today anyhow.

File: tasks-by-users/nehalskdev/02 Poject/index.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta charset="name" content="internet, google, youtube, blog, vlog">
```
Comment (shaikh-7abish) [inline tasks-by-users/nehalskdev/02 Poject/index.html:5]: This should be **name="keywords"**

> not charset="name"

File: tasks-by-users/nehalskdev/02 Poject/index.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    <titl> <h1>Airborne Traveler ©</h1> </titl>
```
Comment (shaikh-7abish) [inline tasks-by-users/nehalskdev/02 Poject/index.html:6]: `title`

>you don't have to use h1 in title

File: tasks-by-users/nehalskdev/02 Poject/Thanks.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="Thanks, Appreciation">
```
Comment (shaikh-7abish) [inline tasks-by-users/nehalskdev/02 Poject/Thanks.html:5]: what is the use of this ?

File: tasks-by-users/nehalskdev/02 Poject/index.html (hunk 3 above)
Comment (nehalskdev) [reply tasks-by-users/nehalskdev/02 Poject/index.html]: corrected as per your comment

File: tasks-by-users/nehalskdev/02 Poject/Thanks.html (hunk 5 above)
Comment (nehalskdev) [reply tasks-by-users/nehalskdev/02 Poject/Thanks.html]: it came with shift+! i forgot to remove. now i have removed it as per your comment

File: tasks-by-users/nehalskdev/02 Poject/index.html (hunk 4 above)
Comment (nehalskdev) [reply tasks-by-users/nehalskdev/02 Poject/index.html]: gotcha. i have made changes as per your sugguestions please check and approve. thanks

**Pull request merged**